site/docs/config-file.mdx

```diff
@@ -85,7 +120,3 @@ These plugins will run after once all presets operations are complete.
   }
   ```
 
-## Alternatives
```

Comment:
The section about the PostCSS config file has been moved to the top as that is what most people will use and the PostCSS config overrides the config in the cssnano files.